File: app_sample_java/src/main/java/com/joelzhu/lib/scanner/java/commands/CommandWork.java
package com.joelzhu.lib.scanner.java.commands;

import com.joelzhu.lib.scanner.annotation.CompileScan;
import com.joelzhu.lib.scanner.java.ICommand;
import com.joelzhu.lib.scanner.java.utils.Constants;

/**
 * Command: man to work.
 *
 * @author JoelZhu
 * @since 2023-03-10
 */
@CompileScan(tag = Constants.TAG_MAN, group = Constants.GROUP_WORK, priority = 1)
public final class CommandWork implements ICommand {
    @Override
    public String execute() {
        return "Working... By man, priority = 1";
    }
}

File: app_sample_multimodule_app/src/main/java/com/joelzhu/lib/scanner/multimodule/CommandWork.kt
package com.joelzhu.lib.scanner.multimodule

import com.joelzhu.lib.scanner.annotation.CompileScan

/**
 * Command: man to work.
 *
 * @author JoelZhu
 * @since 2023-03-13
 */
@CompileScan(priority = 7)
class CommandWork : ICommand {
    override fun execute(): String {
        return "Working... From app, priority = 7"
    }
}